src/styles/main.scss

```diff
@@ -1,3 +1,202 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-accent-blue: #00acdc;
+
+h3 {
+  font-weight: normal;
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__icon {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  list-style: none;
+  font-weight: 500;
+  font-size: 12px;
+  height: 100%;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 23px 0;
+  color: $color-main-accent;
+}
+
+.nav__link:hover {
+  color: $color-accent-blue;
+}
+
+.is-active {
+  color: $color-accent-blue;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  height: 4px;
+  width: 100%;
+  background: $color-accent-blue;
+  border-radius: 8px;
+}
+
+.content {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.content__row {
+  display: flex;
+  margin: 46px auto 0;
+}
+
+.content__row:first-child {
+  margin-top: 50px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  width: 200px;
+  padding: 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-position: 19px 32px;
+  background-repeat: no-repeat;
+}
+
+.card:not(:last-child) {
+  margin-right: 48px;
```

Comment:
Ideally, BEM block should not know anything about its position. Only elements should have margins, positions, etc.
So, you can use an element like `class="card content__card` for example.

Reply:
not fixed
<img width="762" alt="image" src="https://user-images.githubusercontent.com/53611910/206906530-3ff54187-d769-4d75-b9da-d4d21dab956b.png">